client/app/vps/modal/shortcut/kvm/novnc/novnc.controller.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable import/no-unresolved */
+import RFB from '@novnc/novnc';
+/* eslint-enable import/no-unresolved */
+
+import {
+  STATUS_CONNECTING,
+  STATUS_CONNECTED,
+  STATUS_DISCONNECTED,
+} from './novnc.constants';
+
+export default class DedicatedVpsKVMNoVNCController {
+  $onInit() {
+    this.connected = false;
+    this.desktopName = '';
+    this.status = STATUS_CONNECTING;
+
+    const rfbUrl = `wss://${this.host}:${this.port}/`;
+
+    const rfbConfig = {
+      credentials: {
+        password: this.password,
+      },
+      wsProtocols: ['binary', 'base64'],
+      shared: true,
+      repeaterID: '',
+    };
+
+    this.rfb = new RFB(
+      document.getElementById('noVNC_screen'),
+      rfbUrl,
+      rfbConfig,
+    );
+
+    this.rfb.addEventListener('connect', () => {
```

Comment:
```suggestion
    this.rfb.addEventListener(EVENTS.connect, () => {
```

Reply:
Constants should be provided by `@novnc/novnc`.